src/Validation.ContentScan.Core/ContentScanMessageSerializer.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using NuGet.Services.ServiceBus;
+
+namespace NuGet.Jobs.Validation.ContentScan
+{
+    public class ContentScanMessageSerializer : IBrokeredMessageSerializer<ContentScanData>
+    {
+        private const string StartScanSchemaName = "StartContentScanData";
+        private const string CheckScanStatusSchemaName = "CheckContentScanStatusData";
+
+        private IBrokeredMessageSerializer<StartContentScanData1> _startScanSerializer =
+            new BrokeredMessageSerializer<StartContentScanData1>();
+        private IBrokeredMessageSerializer<CheckContentScanStatusData1> _checkScanStatusSerializer =
+            new BrokeredMessageSerializer<CheckContentScanStatusData1>();
+        public ContentScanData Deserialize(IBrokeredMessage message)
```

Comment:
Please add a new line here:

```suggestion

        public ContentScanData Deserialize(IBrokeredMessage message)
```